release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -881,6 +881,28 @@ In {product-title} {product-version}, you can use the {factory-prestaging-tool}
 
 You can now configure OS-level tuning for nodes in a hosted cluster by using the Node Tuning Operator. To configure node tuning, you can create config maps in the management cluster that contain `Tuned` objects, and reference those config maps in your node pools. The tuning configuration that is definied in the `Tuned` objects is applied to the nodes in the node pool. For more information, see xref:../scalability_and_performance/using-node-tuning-operator.adoc#node-tuning-hosted-cluster_node-tuning-operator[Configuring node tuning in a hosted cluster].
 
+[id="ocp-4-12-nfd-multi-namespaces"]
+==== Node Feature Discovery supports multiple namespaces
+
+The Node Feature Discovery (NFD) Operator now supports multiple namespaces.
+
+[id="ocp-4-12-kmmo"]
+==== Kernel Module Management Operator
+
+The Kernel Module Management Operator (KMMO) replaces the Special Resource Operator (SRO). 
```

Comment:
We are not allowed to capitalize kernel module management.  It must be lower case.